package.json

```diff
@@ -105,7 +105,8 @@
     "fast-xml-parser": "4.2.5",
     "semver": "7.5.2",
     "tough-cookie": "4.1.3",
-    "aws-cdk-lib": "2.80.0"
+    "aws-cdk-lib": "2.80.0",
+    "axios": "1.6.0"
```

Comment:
Do we need this resolution? I don't think anything is depending on an older version of Axios so shouldn't we just be able to upgrade the version we are using in the dependecies above?

Reply:
I'm seeing the below:
```
"@aws-amplify/api-rest@3.5.5":
  version "3.5.5"
  resolved "https://registry.yarnpkg.com/@aws-amplify/api-rest/-/api-rest-3.5.5.tgz#2cce71c82b0925f5ea859ba0769a0405130cc5c6"
  integrity sha512-tGR5yLoIC0gPcI8VyAbd7dZ8GdFMz/EEU7aG0HsAsg46Oig5VTtKa8xWV8w+dWXjzr9I2/jkpZtDfRD57PqiBg==
  dependencies:
    "@aws-amplify/core" "5.8.5"
    axios "0.26.0"
    tslib "^1.8.0"
    url "0.11.0"
```

Reply:
oh yes you are correct, sorry didn't see that